releasenotes/notes/add-language-detection-client-33daf0cbadb92d59.yaml

```
@@ -0,0 +1,9 @@
+features:
+  - |
+    The Core-Agent now transmits language metadata to the Cluster-Agent when ``language_detection.enabled`` is set to ``true``. 
```

Feel free to dismiss this suggestion if we are referencing to Agent 5, I found one instance of the "Core Agent" on the Docs Site in https://docs.datadoghq.com/agent/versions/?tab=agentv6vsv5. 

```suggestion
    The Datadog Agent now transmits language metadata to the Cluster Agent when ``language_detection.enabled`` is set to ``true``. 
```